src/lib/Microsoft.Health.Events/EventHubProcessor/EventProcessor.cs

```diff
@@ -29,12 +29,29 @@ public override async Task RunAsync(CancellationToken ct)
             EventProcessorClient.ProcessErrorAsync += ProcessErrorHandler;
             EventProcessorClient.PartitionInitializingAsync += ProcessInitializingHandler;
 
-            try
+            // Try to connect and read events
```

Comment:
One thing that I forgot to mention: I don't think that we use this class in PaaS. We use this [class](https://dev.azure.com/microsofthealth/Health/_git/workspace-platform?path=/iot-connector/core/iomt-fhir/Microsoft.Health.Fhir.Ingest.Console.PaaS/ResumableEventProcessor.cs), which extends `BaseEventProcessor`. So we might also need these changes in there.

Reply:
yeah I will update it there when this is merged